packages/manager/modules/pci/src/projects/project/storages/object-storage/objects/objects.routing.js

```diff
@@ -24,6 +24,10 @@ export default /* @ngInject */ ($stateProvider) => {
         archive,
         projectId,
       ) => PciProjectStorageContainersService.getAll(projectId, archive),
+
+      containersRegions: /* @ngInject */ (containers) =>
```

Comment:
Nitpick: this block is duplicated in multiple files packages/manager/modules/pci/src/projects/project/storages/object-storage/object-storage.routing.js
is it possible to put in parent route and reuse?